dashboard/src/components/searchresults/SearchResultsView.js

```diff
@@ -112,42 +113,39 @@ const SearchResultsList = ({ type, contents, setVariant }) => {
 };
 
 var searchTypeMap = {
-  "FEATURE": "Feature", //feature variant
-  '\u0005': "Label",
-  '\u0006': "TrainingSet",
-  '\u0007': "Source",
-  '\u0008': "Provider",
-  '\u0009': "Entity",
-  '\u000a': "Model",
-  '\u000b': "User",
+  "FEATURE": "Feature",
+  "FEATURE_VARIANT": "Feature",
+  "LABEL": "Label",
+  "LABEL_VARIANT": "Label",
+  "TRAINING_SET": "TrainingSet",
+  "TRAINING_SET_VARIANT": "TrainingSet",
+  "SOURCE": "Source",
+  "SOURCE_VARIANT": "Source",
+  "PROVIDER": "Provider",
+  "ENTITY": "Entity",
+  "MODEL": "Model",
+  "USER": "User",
 }
```

Comment:
@sdreyer for the purposes of search, do variant mappings point back to the original type?

Reply:
Not sure I understand the question but you also have to think about the situation where I'm searching for a specific variant. we also don't want to clog up a search result with all of the variants if I'm searching for a feature.

Ex.
If I search for "abc"
And I have a feature variant with name F1 and Variants [abc, def, ghi], then the abc variant should show up in the results.

Reply:
Gotcha, that makes sense to me. After the user clicks a variant search result, the existing code has a "hasVariant" flag that handles the routing. All clear!